bcipy/simulator/README.md

```diff
@@ -0,0 +1,70 @@
+## RSVP Simulator
+
+### Overview
+
+This Simulator module aims to automate experimentation by sampling EEG data from prior sessions and running given models in a task loop, thus simulating a live session.
+
+### Run steps
+
+`main.py` is the entry point for program. After following `BciPy` readme steps for setup, run the module from terminal:
+
+```
+(venv) $ python bcipy/simulator -h
```

Comment:
We would want to document other clients here or in a linked README (ex. helpers/state). Otherwise, we could move those clients to a demo if they will not be used often.